haystack/components/converters/xlsx.py

```diff
@@ -22,6 +20,9 @@
 with LazyImport("Run 'pip install tabulate'") as tabulate_import:
     from tabulate import tabulate  # pylint: disable=unused-import # the library is used but not directly referenced
 
+with LazyImport("Run 'pip install pandas'") as pandas_import:
+    import pandas as pd
+
```

Comment:
Could we combine this LazyImport with the openpyxl one? `openpyxl` is needed by pandas to be able to load an excel file. 

We can leave tabulate separately since it's only needed if markdown export is selected instead of csv.

Reply:
thanks. Done!